azure/packages/azure-client/src/utils.ts

```diff
@@ -19,9 +34,68 @@ export function isAzureRemoteConnectionConfig(
 
 /**
  * Type guard for validating a given AzureConnectionConfig is a local connection type (AzureLocalConnectionConfig)
+ *
+ * @public
+ * @remarks
+ * Use this function to confirm if a `AzureConnectionConfig` is a local connection type.
+ * @param connectionConfig - The configuration object to check.
+ * @returns True if the given config is an local connection, otherwise false.
+ * @example
+ * ```typescript
+ * const config: AzureConnectionConfig = { type: 'local', ... };
+ * if (isAzureLocalConnectionConfig(config)) {
+ *     console.log('This is a local connection config.');
+ * }
+ * ```
+ * @see {@link AzureConnectionConfig}
+ * @see {@link AzureLocalConnectionConfig}
  */
 export function isAzureLocalConnectionConfig(
 	connectionConfig: AzureConnectionConfig,
 ): connectionConfig is AzureLocalConnectionConfig {
 	return connectionConfig.type === "local";
 }
+
+/**
+ * Test API with comprehensive TSDoc documentation.
+ *
+ * @remarks
+ * This API is designed to showcase all possible features of TSDoc and API Extractor.
+ *
+ * @defaultValue 'N/A'
+ * @decorator TEST
+ * @deprecated Use another API instead.
+ * @eventProperty
+ * @example
+ * ```typescript
+ * const result = testTSDocBellsAndWhistles<string>();
+ * ```
+ * @example
```

Comment:
I recently added support for example titles to our api-documenter library. I haven't quite integrated it into the website build yet, but in preparation (and to better conform to spec) I would recommend updating the lines with the `@example` tags to say `@example TypeScript` and `@example JavaScript` before the following sample code.

Reply:
PR (for reference): #17207